packages/connectivity/pubspec.yaml

```diff
@@ -26,4 +26,4 @@ dev_dependencies:
 
 environment:
   sdk: ">=2.0.0-dev.28.0 <3.0.0"
-  flutter: ">=1.6.7 <2.0.0"
+  flutter: ">=1.6.7 <2.0.0"
```

Comment:
Does this really work with Flutter 1.6.7 or does it need a higher minimum bound? If I'm remembering right a lot of the federated plugin work is newer, right?

Reply:
AFAIK this change is not using any new federated plugin feature. /cc @amirh  is that right?

Reply:
The macOS plugin uses the new pubspec declaration format, but if the dependency on the macOS plugin isn't being added yet the version requirement of the main plugin doesn't change AFAIK.